storage/libradb/src/ledger_store/mod.rs

```diff
@@ -21,6 +21,7 @@ use libra_crypto::{
     hash::{CryptoHash, TransactionAccumulatorHasher},
     HashValue,
 };
+use libra_logger::prelude::*;
```

Comment:
Can you remove this line and land this change? I want to see how cluster-test numbers improve with this PR